ps/jenkins/percona-server-for-mysql-8.0.groovy

```diff
@@ -207,11 +203,123 @@ def package_tests_ps80(def nodes) {
     }
     parallel stepsForParallel
 }
+def docker_test() {
+    def stepsForParallel = [:] 
+        stepsForParallel['Run for ARM64'] = {
+            node('docker-32gb-aarch64') {
+                stage("Docker image version check for ARM64") {
+                    script{
+                        sh '''
+                            echo "running test for ARM"
+                            export DOCKER_PLATFORM=linux/arm64
+                            # disable THP on the host for TokuDB
+                            echo "echo never > /sys/kernel/mm/transparent_hugepage/enabled" > disable_thp.sh
+                            echo "echo never > /sys/kernel/mm/transparent_hugepage/defrag" >> disable_thp.sh
+                            chmod +x disable_thp.sh
+                            sudo ./disable_thp.sh
+                            # run test
+                            export PATH=${PATH}:~/.local/bin
+                            sudo yum install -y python3 python3-pip
+                            rm -rf package-testing
+                            git clone https://github.com/Percona-QA/package-testing.git --depth 1
+                            cd package-testing/docker-image-tests/ps-arm
+                            pip3 install --user -r requirements.txt
+                            export PS_VERSION="${PS_RELEASE}-arm64"
+                            echo "printing variables: \$DOCKER_ACC , \$PS_VERSION , \$PS_REVISION "
+                            ./run.sh
+                            docker run -dit -e MYSQL_ROOT_PASSWORD=asdasd --name mysqlcontainer ${DOCKER_ACC}/percona-server:${PS_VERSION}
+                            fetched_docker_version=$(docker exec mysqlcontainer bash -c "mysql --version" | awk '{print $3}')
```

Comment:
docker run -it -e MYSQL_ROOT_PASSWORD=asdasd --name mysqlcontainer percona/percona-server:8.0 bash -c "mysql --version" | awk '{print $3}'